docs/demo/gaiden-css/components/dialog/aligned-left/demo.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+  <link rel="stylesheet" href="/gaiden-css/gaiden.css">
+  <link rel="stylesheet" href="/gaiden-css/base.css">
+</head>
+<body>
+  <section class="container wall content-centered col-centered">
+    <div class="dialog dialog--aligned-left col-small-12 col-normal-5 space-box-large">
+      <blockquote class="dialog__quote">
+        <p>"Precisava fazer um telhado no meu quintal, entrei no GetNinjas e fiz um pedido de pedreiro, eles entraram em contato comigo no mesmo dia, escolhi o que melhor me atendeu e fechei o serviço."</p>
+      </blockquote>
+
+      <div class="dialog__user space-box-small">
```

Comment:
@MarcoBrunoBR This change will break the existing layout, I think we could talk about semantics on another PR, @eduardojmatos what do you think?